README.md

```diff
@@ -150,6 +150,10 @@ ansible-playbook clean-up.yml
 
 * Make sure your GCP Project Name is set.  To set it, go to *API Manager &gt; Credentials &gt; OAuth consent screen*.
 
+* If Ansible keeps saying that requests or googleauth aren't installed, run `which python` to see which copy of the
```

Comment:
I had this issue on my Mac. I think it's a Homebrew (Mac package manager) thing, but I'll include it nonetheless.